website/integrations/services/apache-guacamole/index.mdx

```diff
@@ -89,7 +89,7 @@ Additionally, ensure your `guacamole.properties` file (typically located in `/et
     openid-client-id=<Client ID from authentik>
     openid-issuer=https://authentik.company/application/o/<your-slug>/
     openid-jwks-endpoint=https://authentik.company/application/o/<your-slug>/jwks/
-    openid-redirect-uri=https://guacamole.company/ # This must match the Redirect URI set in authentik (Including trailing slash).
+    openid-redirect-uri=https://guacamole.company/.
```

Comment:
```suggestion
    openid-redirect-uri=https://guacamole.company/
```

Reply:
Oooof good catch on that left-over punctuation, thanks @BeryJu !

I think we should leave the helpful Notes about making sure that the value matches the Redirect URI, and that the trailing slash is important. Why would we NOT want to include that helpful info? (Though we do need to make them match.)